streaming/src/main/scala/org/apache/carbondata/streaming/StreamHandoffRDD.scala

```diff
@@ -277,15 +277,21 @@ object StreamHandoffRDD {
    */
   def startStreamingHandoffThread(
       carbonLoadModel: CarbonLoadModel,
-      sparkSession: SparkSession
+      sparkSession: SparkSession,
+      isDDL: Boolean
   ): Unit = {
     // start a new thread to execute streaming segment handoff
     val handoffThread = new Thread() {
```

Comment:
move this code to the inside of the if statement

Reply:
Done